extensions/binding/org.eclipse.smarthome.binding.mqtt.generic/src/main/java/org/eclipse/smarthome/binding/mqtt/generic/internal/MqttChannelTypeProvider.java

```diff
@@ -73,15 +72,11 @@ public void removeChannelGroupType(ChannelGroupTypeUID uid) {
         groups.remove(uid);
     }
 
-    public void addChannelGroupType(ChannelGroupTypeUID uid, ChannelGroupType type) {
+    public void setChannelGroupType(ChannelGroupTypeUID uid, ChannelGroupType type) {
```

Comment:
The underlying container is a Map. "add" implies that I append the type, but I'm actually "putting" it in the map or "setting" it. I had changed this after I've seen myself writing code like this:
```
removeChannelGroupType(uid);
addChannelGroupType(uid, type);
```
which is of course not necessary, because the "add" call would replace the value.